tests/test_integration_scenarios.py

```
@@ -56,12 +56,52 @@ def _reset_global_state(self):
+                            module,
+                            "_disconnect_ble_interface",
+                            None,
+                        )
+                        if callable(disconnect_iface):
+                            disconnect_iface(iface, reason="test-reset")
+                        else:
+                            close = getattr(iface, "close", None)
+                            if callable(close):
+                                close()
+                    except Exception:
+                        pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)